src/Button/helpers.js

```diff
@@ -0,0 +1,129 @@
+import { css } from 'styled-components';
+import { darken } from 'polished';
+
+/**
+ * getAppearance
+ *
+ * Helper function to render the right css
+ * compared to the appearance given to him.
+ *
+ * @param {object} palette // The pallet color theme.
+ * @param {string} appearance // The button appearance.
+ */
+
+export const getAppearance = ({ palette }, appearance) => {
```

Comment:
why not just passing `(palette, appareance)` instead of passing all the theme in this method ? if you only use `palette` in this method I don't understand why you passing the theme and why you destructure `palette`

Reply:
I desctructure `palette` because I juste want this `object`. I passing theme because that's what give me styled component in `src/Button/index`

Reply:
@spartDev I'm not agree. You don't need `Theme` here, it's simply an template string that will be used to define Button style.
And you are using : `${({ theme: { palette } }) => palette.whiteOpacity(0)} 0%,`. You should simply use `palette.whiteOpacity` instead.